Casks/b/buzz.rb

```diff
@@ -1,14 +1,15 @@
 cask "buzz" do
-  version "1.1.0"
-  sha256 "9ca6ca83d2b3b2cdf9b01ae396c6bf990530b5c9531de4a1793999e64d83e071"
+  arch arm: "arm64", intel: "x64"
 
-  url "https://github.com/chidiwilliams/buzz/releases/download/v#{version}/Buzz-#{version}-mac.dmg"
+  version "1.2.0"
+  sha256 arm:   "8bc210ead8ff0f694246f772ff48b8f0d777d4c5df2594cac57937a5fc0f61d1",
+         intel: "72bd99f2e80fcbb4b29e440bc0e7a471472b14a68d86a65c7fa0857c6453ef03"
+
+  url "https://github.com/chidiwilliams/buzz/releases/download/v#{version}/Buzz-#{version}-mac-#{arch}.dmg"
   name "Buzz"
-  desc "Transcribes and translates audio"
+  desc "Buzz transcribes and translates audio offline on your personal computer. Powered by OpenAI's Whisper."
```

Comment:
```suggestion
  desc "Transcribes and translates audio"
```

The existing `desc` conforms better to the [stanza guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc).